src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -5508,9 +5508,8 @@ public final class TokenTypes {
      * Beginning of single-line comment: '//'.
      *
      * <pre>
-     * +--SINGLE_LINE_COMMENT
-     *         |
-     *         +--COMMENT_CONTENT
+     * --SINGLE_LINE_COMMENT -&gt; //
```

Comment:
Please remove `--` to make whole content to move left and take less horizontal space.

Reply:
Done